src/diag.rs

```diff
@@ -4,8 +4,14 @@
 //! interpretation and testing, as well as a mostly-text representation which
 //! can be used for various human-readable outputs.
 
-use parser::{Comparer, Span, StatementIndex, StatementAddress, StatementRef, Token, TokenIndex,
-             TokenAddress};
+use parser::Comparer;
+use parser::Span;
+use parser::StatementAddress;
+use parser::StatementIndex;
+use parser::StatementRef;
+use parser::Token;
+use parser::TokenAddress;
+use parser::TokenIndex;
```

Comment:
Note that this commit says "revert rejected changes" but it is actually "reverting" a change that I did not introduce - you said you didn't like horizontal imports so I ungrouped all of them, including this group which was there when I started.